model/src/commonMain/kotlin/VulnerabilityRating.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.model
+
+/**
+ * The overall rating for a vulnerability. The rating is derived from the severities of the individual references of a
+ * vulnerability as the highest severity found, that matches one of the [VulnerabilityRating] values. If no such
+ * severities are found, the rating is [NONE].
+ * */
+enum class VulnerabilityRating(val score: Int) {
```

Comment:
"score" probably is a bit of a misleading variable name here... the value seems to be more used like an ordinal value for the enum, which does not need to be stored explicitly, as it's available built-in like `CRITICAL.ordinal`. Just reorder enum entries so that `NONE` gets 0 assigned implicitly.